DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/AccessibilityTestsBase.cs

```diff
@@ -1,28 +1,21 @@
-﻿namespace DigitalLearningSolutions.Web.AutomatedUiTests.AccessibilityTests
+namespace DigitalLearningSolutions.Web.AutomatedUiTests.AccessibilityTests
 {
-    using System;
-    using DigitalLearningSolutions.Web.AutomatedUiTests.TestHelpers;
+    using DigitalLearningSolutions.Web.AutomatedUiTests.TestFixtures;
     using FluentAssertions;
     using OpenQA.Selenium;
     using Selenium.Axe;
     using Xunit;
 
     [Collection("Selenium test collection")]

```

Comment:
All the accessibility tests are in one collection so that they don't run in parallel.

Reply:
You deleted the file with the [CollectionDefinition] that was used to do that. I sit no longer necessary?

Reply:
I added them at the same when I was adding new files (for the journeys rather than basic) to force them to not run in parallel. I guess your new fix removed the need for it?

See Collection Fixtures here for some documentation https://xunit.net/docs/shared-context

Reply:
The CollectionDefinition before had the SeleniumServerFactory collection fixture, which just meant the same server factory was reused for all tests in the collection. But that doesn't make a difference anyway. What we're doing now is each test class has a class fixture which makes a new driver (and new serverfactory but that's not important) at the beginning and reuses the same driver across all tests.
We do need all the test classes to be in the same collection so that they don't run in parallel. To achieve that you just need Collection, you don't need CollectionDefinition.